shaptools/shapcli.py

```diff
@@ -0,0 +1,353 @@
+"""
+Code to expose some useful methods using the command line
+
+:author: xarbulu
+:organization: SUSE LLC
+:contact: xarbulu@suse.com
+
+:since: 2019-07-11
+"""
+
+import logging
+import argparse
+import json
+
+from shaptools import hana
+
+PROG = 'shapcli'
+LOGGING_FORMAT = '%(message)s'
+
+
+class DecodedFormatter(logging.Formatter):
```

Comment:
do we need a Class here? I mean imho I would just use a helper function without the `super` overloading if possible.